src/System.Runtime.Extensions/tests/AssemblyResolveTests/AssemblyResolveTests.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <ProjectGuid>ad83807c-8be5-4f27-85df-9793613233e1</ProjectGuid>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <CopyNuGetImplementations>false</CopyNuGetImplementations>
```

Comment:
Agreed, I think we don't ever want this to be true, so I'll put it in dir.props.